ui/src/components/EmailForm.vue

```diff
@@ -78,27 +75,125 @@
       <v-textarea :label="$t('groups.members.email.body')" v-model="email.body">
       </v-textarea>
     </v-card-text>
+    <v-flex class="text-xs-center">
+      <v-btn
+        class="ma-2"
+        flat
+        small
+        color="primary"
+        @click="showEntityTypePanel"
+        :disabled="entityTypePanel.show"
+      >
+        {{ $t('actions.tooltips.settings') }}
+      </v-btn>
+    </v-flex>
+    <v-flex>
+      <!-- extention for email     -->
+      <v-expand-transition>
+        <v-card
+          v-if="entityTypePanel.show"
+          color="teal lighten-3"
+        >
+          <v-radio-group v-model="radioGroup">
+            <v-card-title>{{ $t('groups.members.email-reply-to') }} :   {{ radioGroup || 'null' }}</v-card-title>
+            <v-card-title text color="green"></v-card-title>
+            <v-radio
+              :label= "$t('groups.title')"
+              :key='1'
+              :value=homeChurchEmail
+            ></v-radio>
+              <v-radio
+                :label= "$t('groups.details.manager')"
+                @click="showManagerPanel"
+                :key='2'
+                :value = replyToOtherEmail
+              ></v-radio>
+              <v-expand-transition>
+                <v-card
+                  v-if="managerPanel.show"
+                >
+                  <v-card-text>
+                    <entity-search
+                      multiple
+                      person
+                      v-model="selectedPerson"
+                      :existing-entities="searchPeople"
+                    />
+<!--                    :existing-entities="attendance"-->
+                  </v-card-text>
+                  <v-btn v-on:click="hideManagerPanel" color="light-blue" flat>{{
+                    $t("actions.cancel")
+                    }}</v-btn>
+                  <v-btn v-on:click="setReplyTo" color="primary" flat>{{
+                    $t("actions.confirm")
+                    }}</v-btn>
+                </v-card>
+              </v-expand-transition>
+<!--            <v-expand-transition>-->
```

Comment:
Do we want to retain this commented-out markup?

Reply:
They should be removed, that was a mistake I made. Sorry about this.